lib/cadet_web/router.ex

```diff
@@ -210,6 +210,12 @@ defmodule CadetWeb.Router do
   #   pipe_through :api
   # end
 
+  scope "/v2", CadetWeb do
+    pipe_through(:api)
+
+    resources("/shared_programs", SharedProgramController, only: [:index, :show, :create, :delete])
```

Comment:
Are we supporting `:index` and `:delete`? Only create and read operations are meant to be supported right?